### ISSUE TITLE
Make kraftwagen available on packagist.org

### ISSUE THREAD

**Issue by vladdancer** [kraftwagen/kraftwagen#52]
Could you please add kraftwagen as a package to the packagist.org.
We want to install it as global package inside vagrant box to build our project.
Some example can be found here https://github.com/drush-ops/config-extra/blob/master/composer.json

**Comment by bighappyface** [comment]
I install drush and kraftwagen via composer.

Here is the snippet:

```json
"require": {
	"drush/drush": "6.6.0"
},
"scripts": {
	"post-install-cmd": [
	  "test -d vendor/drush/drush/commands/kraftwagen || git clone https://github.com/kraftwagen/kraftwagen.git vendor/drush/drush/commands/kraftwagen",
	  "bin/drush cc drush"
	]
}
```

It's by no means ideal, and doesn't work on Windows, but it works for me.

**Comment by vladdancer** [comment]
Thanks, I saw this solution in another task. But It's not useable for us. And anyway an adding composer.json is a good idea because we have stable drupal8 + many drush extensions has composer.json inside. De-facto every open source php library should be registered on packagist.

Thanks.

**Comment by rolfvandekrol** [comment]
Fixed by 6480bfbf3385c7e9d1ff8e7fe6f8cdff6ca9217e

**Comment by vladdancer** [comment]
Thanks, Rolf!
